project/src/components/offers-favorites-list/offers-favorites-list.tsx

```diff
@@ -1,51 +1,48 @@
-import {useEffect} from 'react';
-import {useDispatch, useSelector} from 'react-redux';
-import {fetchFavoritesAction} from '../../store/api-actions';
-import {getFavoriteOffers} from '../../store/offers/selectors';
+import {Offer} from '../../types/offer';
 import {getCityOffers} from '../../utils/utils';
 import {OfferFavoriteCard} from '../offer-favorite-card/offer-favorite-card';
 
-function OffersFavoritesList(): JSX.Element{
-  const dispatch = useDispatch();
-  const offers = useSelector(getFavoriteOffers);
+type OffersFavoritesListProps = {
+  offers: Offer[] | null;
```

Comment:
```suggestion
  offers: Offer[];
```
используй пустой массив